### PR TITLE
[#12466] Clarify masquerade mode in docs

### LINKED ISSUE #12466: Clarify masquerade mode in docs
The [docs](https://teammates.github.io/teammates/development.html#logging-in-to-a-teammates-instance) on masquerade mode is unclear on what username is on the development server 

`When logged in as administrator, masquerade mode can also be used to impersonate instructors and students by adding user=username to the URL e.g http://localhost:8080/web/student/home?user=johnKent.`

Developers who want to use masquerade mode have to ensure that the`username`  is the email address used to verify the user e.g (person@x.com) when signing up mock instructors. Username such as jack (for google id jack@example.com) would not work as the username.

image
Signed up jack@example.com as instructor, which is also used as google id

image
Incorrect username used (jack)

image
Correct username used (jack@example.com)

### PR DESCRIPTION
**Pull request by JacksonsProfileViewer**
Fixes #12466

**Outline of Solution**

 

Changed the documentation for masquerade mode. Made it clear that the user should be the entire email address and not just the leading portion. ( New to this project so not 100% if it can't be something that is not an email. )

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -128,7 +128,7 @@ This instruction set applies for both dev server and production server, with sli
   You will be granted access only if your account has admin permission as defined in `build.properties`.
 
 1. When logged in as administrator, ***masquerade mode*** can also be used to impersonate instructors and students by adding `user=username` to the URL
-  e.g `http://localhost:8080/web/student/home?user=johnKent`.
+  e.g `http://localhost:8080/web/student/home?user=johnKent@example.com` where the user to impersonate has a `username` of `johnKent@example.com`.
```
Comment (domlimm) [inline docs/development.md:131]: Let's change it to:
e.g `http://localhost:8080/web/student/home?user=johnKent@example.com`.

File: docs/development.md [hunk 2]
```diff
@@ -128,7 +128,7 @@ This instruction set applies for both dev server and production server, with sli
   You will be granted access only if your account has admin permission as defined in `build.properties`.
 
 1. When logged in as administrator, ***masquerade mode*** can also be used to impersonate instructors and students by adding `user=username` to the URL
```
Comment (domlimm) [inline docs/development.md:130]:
```suggestion
1. When logged in as administrator, ***masquerade mode*** can also be used to impersonate instructors and students by adding `user=email` to the URL. Developers who want to use masquerade mode have to ensure that the username is the email address used to verify the user, e.g user@email.com, when signing up mock instructors.
```

**Review by domlimm — COMMENTED** [review]
Just some changes. Might have some changes again (following the issue). Let's see.

File: docs/development.md (hunk 1 above)
Comment (domlimm) [reply docs/development.md:131]: GitHub doesn't allow me to suggest. Basically, remove everything from `...where the user to...`. Thanks!

**Review by domlimm — COMMENTED** [review]
One final change. Other than that LGTM! Thank you for your contribution!

**Comment by weiquu** [comment]
Hi @JacksonsProfileViewer, do take a look at the requested changes

**Comment by weiquu** [comment]
Closing due to inactivity.

**Pull request closed without merging**